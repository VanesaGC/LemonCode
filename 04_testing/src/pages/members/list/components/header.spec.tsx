import * as React from "react";
import { shallow } from "enzyme";
import { Header } from "./header";

describe("pagesmemberslistcomponents (Header) tests", () => {
  it("should render as expected when is invoked", () => {
    // Arrange

    // Act
    const component = shallow(<Header />);
    // Assert
    expect(component).toMatchSnapshot();
  });
});
