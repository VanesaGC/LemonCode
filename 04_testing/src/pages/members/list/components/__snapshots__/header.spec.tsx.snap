// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests should render as expected when is invoked 1`] = `
<thead>
  <tr>
    <th>
      Avatar
    </th>
    <th>
      Id
    </th>
    <th>
      Name
    </th>
  </tr>
</thead>
`;

exports[`common/components/panel/header tests should render as expected when is invoked 1`] = `
<thead>
  <tr>
    <th>
      Avatar
    </th>
    <th>
      Id
    </th>
    <th>
      Name
    </th>
  </tr>
</thead>
`;

exports[`pagesmemberslistcomponents  tests should render as expected when is invoked 1`] = `
<thead>
  <tr>
    <th>
      Avatar
    </th>
    <th>
      Id
    </th>
    <th>
      Name
    </th>
  </tr>
</thead>
`;

exports[`pagesmemberslistcomponents (Header) tests should render as expected when is invoked 1`] = `
<thead>
  <tr>
    <th>
      Avatar
    </th>
    <th>
      Id
    </th>
    <th>
      Name
    </th>
  </tr>
</thead>
`;

exports[`pagesmemberslistcomponents tests should render as expected when is invoked 1`] = `
<thead>
  <tr>
    <th>
      Avatar
    </th>
    <th>
      Id
    </th>
    <th>
      Name
    </th>
  </tr>
</thead>
`;
